packages/features/bookings/Booker/components/BookEventForm/BookFormAsModal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Calendar, Clock } from "lucide-react";
+
+import dayjs from "@calcom/dayjs";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Badge, Dialog, DialogContent } from "@calcom/ui";
+
+import { useBookerStore } from "../../store";
+import { useEvent } from "../../utils/event";
+import { BookEventForm } from "./BookEventForm";
+
+export function BookFormAsModal({ visible, onCancel }: { visible: boolean; onCancel: () => void }) {
+  const { t } = useLocale();
+  const selectedTimeslot = useBookerStore((state) => state.selectedTimeslot);
+  const selectedDuration = useBookerStore((state) => state.selectedDuration);
+  const { data } = useEvent();
+  const parsedSelectedTimeslot = dayjs(selectedTimeslot);
+
+  return (
+    <Dialog open={visible} onOpenChange={onCancel}>
+      <DialogContent type={undefined} enableOverflow className="max-h-[45vh]">
```

Comment:
Should we limit the height? Because now I see this scrollbar, while it's actually not needed.

![CleanShot 2023-04-25 at 09 17 13@2x](https://user-images.githubusercontent.com/2969573/234202514-8d0bf110-5f49-4e7c-9e04-60c8e6b20193.jpg)